CHAINCODE_AS_A_SERVICE_TUTORIAL.md

```diff
@@ -0,0 +1,197 @@
+# Running Chaincode as Service with the Test Network
```

Comment:
Rather than have in root, I think this MD makes more sense in /test-network directory. You can also put a link to it from /test-network/README.md.
I will also link to it in the release announcement.

Reply:
that's perfectly fine; didn't really seem to be a correct place.